### PR TITLE
(BKR-1528) Use `puppetserver ca` instead of `puppet cert`

### PR DESCRIPTION
**Pull request by Magisus**

### CODE REVIEW

File: lib/beaker-puppet/helpers/puppet_helpers.rb [hunk 1]
```diff
@@ -845,8 +858,15 @@ def sign_certificate_for(host = [])
...
+
+              if version_is_less(puppet_version, '5.99')
+                on master, puppet("cert --sign --all --allow-dns-alt-names"), :acceptable_exit_codes => [0,24]
+                out = on(master, puppet("cert --list --all")).stdout
+              else
+                on master, 'puppetserver ca sign --all', :acceptable_exit_codes => [0, 24]
+                out = on(master, 'puppetserver ca list --all').stdout
+              end
+
               if hostnames.all? { |hostname| out =~ /\+ "?#{hostname}"?/ }
```
Comment (justinstoller) [inline lib/beaker-puppet/helpers/puppet_helpers.rb]: Did we not need to change this after all?

Comment (Magisus) [reply lib/beaker-puppet/helpers/puppet_helpers.rb]: Oh right. Tests are running now. I'll let you know.

Comment (Magisus) [reply lib/beaker-puppet/helpers/puppet_helpers.rb]: Kinda looks like this test is just for efficiency. So it may "work" even if the check is failing.

**Comment by Magisus** [comment]
Updated with new regex for 6.0.

**Pull request merged**